cypress/integration/ws.spec.js

```diff
@@ -0,0 +1,90 @@
+import { WebSocket } from '@graasp/websockets/test/mock-client';
+import { buildMainPath } from '../../src/config/paths';
+import {
+  buildFolderButtonId,
+  FOLDER_NAME_TITLE_CLASS,
+} from '../../src/config/selectors';
+import { FOLDER_WITH_SUBFOLDER_ITEM } from '../fixtures/items';
+import { CURRENT_USER } from '../fixtures/members';
+import { expectFolderButtonLayout } from '../support/integrationUtils';
+import { mockGetChildren, mockGetItem } from '../support/server';
+
+describe('Websocket interactions', () => {
+  let client;
+  const { items } = FOLDER_WITH_SUBFOLDER_ITEM;
+  const newChild = {
+    ...FOLDER_WITH_SUBFOLDER_ITEM.items[1],
+    id: 'deadbeef-aaaa-bbbb-cccc-0242ac130002',
+    name: 'newChild',
+  };
+
+  beforeEach(() => {
+    client = new WebSocket();
+    mockGetItem({ items, currentMember: CURRENT_USER }, false);
+    mockGetChildren(items);
+  });
+
+  const beforeWs = (visitRoute) => {
+    cy.visit(visitRoute, {
+      onBeforeLoad: (win) => {
+        cy.stub(win, 'WebSocket', () => client);
```

Comment:
Factor out `Websocket`?

Reply:
Not sure it's a good idea in this case: I'm specifically dynamically replacing the global `WebSocket` object from `window`. Wouldn't that make it less readable?